src/browser/tab/mod.rs

```diff
@@ -663,6 +663,28 @@ impl<'a> Tab {
             .call_method(protocol::debugger::methods::GetScriptSource { script_id })?
             .script_source)
     }
+
+    /// Executes javascript
+    ///
+    /// See Devtools protocol doc: https://chromedevtools.github.io/devtools-protocol/tot/Runtime#method-evaluate
```

Comment:
The docs should not just refer to an external syntax.

Reply:
Fixed